docs/release-notes/0.4/release-0.4.md

```diff
@@ -0,0 +1,82 @@
+# ML.NET 0.4 Release Notes
+
+Today we are releasing ML.NET 0.4. During this release we have started
+exploring new APIs for ML.NET that enable functionality that is missing from
+the current APIs. We welcome feedback and contributions to the
+[discussion](https://github.com/dotnet/machinelearning/projects/4). While the
```

Comment:
>discussion [](start = 1, length = 10)

Is it discussion or project? the link opens as a project view.

Reply:
The project is a collection of the relevant issues so it might be a good pointer. Should we change to something else?

Reply:
I'll update to make it clearer.